### ISSUE TITLE
questions about effect of number of positives

### ISSUE THREAD

**Issue by yoyoyoohh** [HobbitLong/SupContrast#43]
Hi, thanks for your nice paper and codes! 
I have a question after reading the paper, in the section "Effect of Number of Positives", the number of positives always contain one positive which is the same sample with the anchor but different  data augmentations, and the remainders are different samples from the same class, have you tried make all the positive from the same sample but different augmentations? Does it improve?

**Comment by zou-longkun** [comment]
You seem to be talking about self-supervised contrastive learning, right? Just like SimCLR

**Comment by yoyoyoohh** [comment]
yes, in a self-supervised manner, but with multiple positives which are derived from the same image.
